### DISCUSSION TITLE
Running prisma edge client locally

### DISCUSSION THREAD

**Discussion by un-focused** [prisma/prisma#16114] [Q&A] [answered]
Hi, I have the edge client running in a serverless environment but locally I don't have a data proxy URL so how can I support both environment?

```
import { PrismaClient } from '@prisma/client/edge'; <--- I want to use this with local db urls
```

**Comment by un-focused** [comment, marked as answer]
I figured out a temporary solution.
See: https://github.com/prisma/prisma/discussions/15318

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
